labs/kubescape-relevancy/values.yaml

```diff
@@ -296,8 +296,8 @@ kubevuln:
 
   image:
     # -- source code: https://github.com/kubescape/kubevuln
-    repository: quay.io/kubescape/kubevuln
```

Comment:
Maybe we should first build the new image

Reply:
yes, this is for @rcohencyberarmor to run it with system tests

Reply:
@dwertent done, and disabled by default